lib/libnfs.c

```diff
@@ -528,14 +528,17 @@ nfs_parse_url(struct nfs_context *nfs, const char *url, int dir, int incomplete)
                         nfs_destroy_url(urls);
                         return NULL;
                 }
+                char *original_server = urls->server;
```

Comment:
Move the declaration to the head of the function. I try not ti intermingle code and variable declarations.

Reply:
`*original_server` moved up to head of function